Style.css

```diff
@@ -0,0 +1,1029 @@
+  font-size: 15px;
+  line-height: 20px;
+
+  /* identical to box height, or 133% */
+
+  /* Neutrals/N0 */
+
+  color: #fff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.btn-ruby-1 {
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 71px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.btn-ruby-1 a {
+  width: 37px;
+  height: 20px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+
+  /* identical to box height, or 133% */
+
+  /* Neutrals/N0 */
+
+  color: #fff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.btn-sub-2 {
+  width: 100%;
+  height: 48px;
+  left: 0;
+  top: 0;
+  border: none;
+  background: #ff6b00;
+}
+
+.btn-sub-2 a {
+  width: 100px;
+  height: 24px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  text-decoration: none;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.about-me {
+  width: 375px;
+  height: 1801px;
+  margin-top: 30%;
+  padding-top: 13%;
+
+  /* Theme 3 / Neutral 1 */
+
+  background-color: #1c1a19;
+  background-image: url("images/img_3.png");
+  background-repeat: no-repeat;
+  background-position: 100% 23%;
+}
+
+.headline-5 {
+  width: 327px;
+  height: 79px;
+  margin-left: 7%;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.text-content-3 {
+  width: 300px;
+  height: 148px;
+
+  /* Theme 3 / Body Reguler 2 */
+
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  margin-top: -10%;
+  line-height: 24px;
+
+  /* or 150% */
+  margin-left: 7%;
+
+  /* Neutrals/N20 */
+
+  color: #f4f5f7;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.btn-sub-3 {
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  border: none;
+  width: 156px;
+  height: 48px;
+  margin-left: 7%;
+
+  /* Theme 3 / Primary 1 */
+
+  background: #ff6b00;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.btn-sub-3 a {
+  width: 132px;
+  height: 24px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+
+  /* identical to box height, or 141% */
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+
+  /* Theme 2 / Neutral 11 */
+
+  color: #fff;
+  text-decoration: none;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```

Comment:
- [ ] Kindly implement the button interactions (enable, hover, etc.) to match the given design.
- Here is the given instruction related to button interactions.
- ![image](https://github.com/MohammadYaser/Portfolio/assets/60083437/adbfb0dd-f104-49f8-a7ca-c703ef3c615f)